pkg/components/enterprise.go

```diff
@@ -15,165 +15,141 @@
 package components
 
 var (
-	
 	ComponentAPIServer = component{
 		Version: "v2.7.0-0.dev-70-gcf61025a",
 		Digest:  "sha256:99001709c7e9079bd41f466ff75bfac87369c84b4096994a04fbceaf3c447c85",
 		Image:   "tigera/cnx-apiserver",
 	}
-	
-	
+
```

Comment:
Is there a reason we have two newlines here? Goimports keeps cleaning them up for me, which I think is  good.

Reply:
That's the way the template file is written. We can change that template.